app/pods/cases/case/subcases/subcase/template.hbs

```diff
@@ -11,8 +11,4 @@
     @onMoveSubcase={{this.refreshSubcases}}
   />
 </div>
-<div class="auk-scroll-wrapper__body">
```

Comment:
The wrapper has been moved to the subroutes of `cases.case.subcases.subcase` because in the overview, we have the pagination at the bottom which we want to render outside of the wrapper. I assume it would be possible to let the pagination element break out of the wrapper instead of moving it to the subroute, but that seemed far too hackish to me, so I went for this solution.